src/utils/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './formatting';
+export * from './commandArgs';
```

Comment:
```suggestion
export * from './args';
```
(after rereviewing content)